src/Uno.Foundation/UriExtensions.cs

```diff
@@ -57,10 +57,14 @@ public static string GetExtension(this Uri uri)
 			return String.Empty;
 		}
 
+		public static bool IsAppData(this  Uri uri)
+		{
+			return uri.Scheme.Equals("ms-appdata", StringComparison.OrdinalIgnoreCase);
+		}
+
 		public static bool IsLocalResource(this Uri uri)
 		{
-			return uri.Scheme.Equals("ms-appx", StringComparison.OrdinalIgnoreCase)
-				|| uri.Scheme.Equals("ms-appdata", StringComparison.OrdinalIgnoreCase);
+			return uri.Scheme.Equals("ms-appx", StringComparison.OrdinalIgnoreCase);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'uri' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5304285)